prac_05/state_names.py

```diff
@@ -2,17 +2,24 @@
 CP1404/CP5632 Practical
 State names in a dictionary
 File needs reformatting
+
+Estimate: 5 minutes
+Actual:   5 minutes
+
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
```

Comment:
i think delete the TODO message will make the code looks better

Reply:
I think the work be finished well.